packages/react-native-win32/src/Libraries/Components/Button/ButtonWin32.tsx

```diff
@@ -65,7 +65,7 @@ export class ButtonWin32 extends React.Component<IButtonWin32Props, IButtonWin32
   };
 
   private _setState = (select: SelectState): void => {
-    const state = this._makeState(SelectState.Selected);
+    const state = this._makeState(select);
```

Comment:
This was a real bug that eslint found. Need to check with folks to understand impact.